docs/get-started/setup-lightdash/connect-project.mdx

```diff
@@ -745,6 +745,22 @@ If you want to change this to be something other than the default `target` defin
 
 To read more about dbt targets, [check out the dbt docs here.](https://docs.getdbt.com/reference/dbt-jinja-functions/target)
 
+#### dbt selector 
+
+You can filter out models in your dbt project that you don't want to see in Lightdash. This is useful if you have a large 
+dbt project and you want to speed up the sync process. Unlike [table selection](https://docs.lightdash.com/guides/adding-tables-to-lightdash#limiting-the-tables-in-lightdash-using-dbt-tags), 
+this selector is applied to the dbt models, so it will skip the entire compilation process for the models that you don't want to see in Lightdash.
```

Comment:
```suggestion
this selector is applied to the dbt models, so it will skip the entire compilation process for the models that you don't want to connect to Lightdash.
```